src/drivers/barometer/bmp388/bmp388.cpp

```diff
@@ -257,7 +248,6 @@ bool
 BMP388::validate_trimming_param()
 {
 	uint8_t crc = 0xFF;
-	uint8_t stored_crc;
```

Comment:
May I ask why you move the variable allocation from the beginning to the function to lower in the function.
It's just a matter of preference but I like to allocate the variables at the beginning of a function.
@dagar  @davids5  @hamishwillee  do we guidelines for this type of code styling?

Reply:
I am an embedded software engineer.
I like to keep the scope of local variables as small as possible.
This is a major idea in software development engineers.

Reply:
The compiler doesn't really decrease the stack usage unless you use `{ }` to indicate the scope. 
Personally I like to have all stack variables on top too easily determine the stack usage of single function.
But again it's a matter of preference hence the question what's the actual PX4 code styling on this.

Reply:
There's no strict style guide for this, but my preference has been wherever possible to declare and define it together, even defaulting to const and never leave open the possibility of something being uninitialized. For various reasons I think it's usually better to keep the scope as limited as possible.

That's an interesting point about stack usage though Peter.